index.html

```diff
@@ -185,7 +185,7 @@ <h1>About Myself</h1>
                 <li>
```

Comment:
![My Logo](https://user-images.githubusercontent.com/93039359/200527393-3a669414-36b4-401f-988a-91635cd053f3.PNG)
![get-in-touch](https://user-images.githubusercontent.com/93039359/200527415-8f9add6c-9fa6-49ed-abb6-e56c52696154.PNG)
![html](https://user-images.githubusercontent.com/93039359/200527421-52c0c908-4e10-40bc-a577-00f031459746.PNG)
![Navlink](https://user-images.githubusercontent.com/93039359/200527430-b907f4e0-e118-4531-bcac-d50ac945929f.PNG)
![see-project-button](https://user-images.githubusercontent.com/93039359/200527432-499c13f2-31c2-469f-a948-e5a2ee20d6d5.PNG)
![technology](https://user-images.githubusercontent.com/93039359/200527436-6a9dde14-bbd5-49b1-bf4d-9b92285ec75c.PNG)

- [ ] You can see in the above images that some of your things in your webpage doesn't have a color contrast of 4.5:1. Kindly make it to make your application accessible for all types of users specially color blind users 😊

Reply:
@HAMMAS-SALEEM, thank you for your comment. If you check the figma design, I made My Logo larger than the figma design. I also followed what I was asked to do with regards to using figma's choice of colors.

Reply:
As the last reviewer suggested kindly fix the color contrast ratio of your buttons. Feel free to change the color and the size text to fulfill this requirement. :pray: :pray: You can use[ this tool ](https://www.aditus.io/button-contrast-checker/)to ensure it.